src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,153 @@
+            if (userName != null && userPassword != null) {
+                // Why didn't suit here such expression "INSERT INTO public." + "\"" + TABLE_NAME + "\"" + "(password, name) "
+                //                                      + "VALUES (" + userPassword + ", " + userName + ");";
+                String sqlQuery = String.format("INSERT INTO \"" + TABLE_NAME + "\"(name, password) VALUES('%s','%s') ", userName, userPassword);
+                statement.executeUpdate(sqlQuery);
+            } else {
+                System.out.println("Wrong input");
+            }
+        }
+    }
+
+    private void dropTableWithName(String user) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String sqlQuery = "DROP TABLE IF EXISTS \"" + user + "\" CASCADE ;";
+            statement.executeUpdate(sqlQuery);
+        }
+    }
+
+    private void createTable(String tableName) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String sqlQuery = "CREATE TABLE public.\"" + tableName + "\"" + LINE_SEPARATOR
+                    + "(" + LINE_SEPARATOR
+                    + "id SERIAL PRIMARY KEY ," + LINE_SEPARATOR
+                    + "name TEXT  NOT NULL," + LINE_SEPARATOR
+                    + "password TEXT  NOT NULL" + LINE_SEPARATOR
+                    + ")";
+            statement.executeUpdate(sqlQuery);
+        }
+    }
+
+    private void showExistingTablesName() throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String sqlQuery = "SELECT" + LINE_SEPARATOR
+                    + "*" + LINE_SEPARATOR
+                    + "FROM" + LINE_SEPARATOR
+                    + "pg_catalog.pg_tables" + LINE_SEPARATOR
+                    + "WHERE" + LINE_SEPARATOR
+                    + "schemaname != 'pg_catalog'" + LINE_SEPARATOR
+                    + "AND schemaname != 'information_schema';";
+            ResultSet rs = statement.executeQuery(sqlQuery);
+            System.out.println("Existing Tables:");
+            if (rs.isBeforeFirst()) {
+                while (rs.next()) {
+                    System.out.println("-" + rs.getString("tablename") + LINE_SEPARATOR);
+                }
+            } else {
+                System.out.println("db is empty" + LINE_SEPARATOR);
+            }
+        }
+    }
+
+    private void createDbConnection() {
+        try {
+            connection = DriverManager.getConnection(DB_CONNECTION_PATH, DB_USER, DB_PASSWORD);
+        } catch (SQLException e) {
+            System.out.println("Connection Failed! Check output console");
+            e.printStackTrace();
+        }
+        if (connection != null) {
+            System.out.println("Connection created" + LINE_SEPARATOR);
+        } else {
+            System.out.println("Failed to make connection!");
+        }
+    }
+
+    private void checkJDBCDriver() {
```

Comment:
I see some mkyong style here))) Logic of this method is a part of createDbConnection(). There's no reason to separate them